components/authz-service/server/v2/rules_property_test.go

```diff
@@ -128,11 +136,11 @@ func TestGetRuleProperties(t *testing.T) {
 	// 		func(reqs projectAndRuleReq) bool {
 	// 			defer testDB.Flush(t)
 
-	// 			_, err := createProjectAndRule(ctx, cl, reqs)
+	// 			_, err := createProjectAndRule(ctx, projectClient, policyClient, reqs)
```

Comment:
There are a lot of commented out tests, but it was just as easy to update this dormant code, too, with requisite changes rather than introduce more tech debt.